### PR TITLE
node-postgres v6.4.0

### PR DESCRIPTION
**Pull request by metainfa**
Update `node-postgres` to version `6.4`  and lock all dependencies (done more for tests). Now all tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,30 +35,30 @@
   },
   "homepage": "https://github.com/langpavel/node-pg-async#readme",
   "dependencies": {
-    "bluebird": "^3.4.6",
-    "debug": "^2.6.3",
-    "pg": "^6.1.4"
+    "bluebird": "3.5.0",
+    "debug": "2.6.3",
+    "pg": "6.4.0"
```
Comment (langpavel) [inline package.json]: Fixed version? Why it is better than `^` range

Comment (metainfa) [inline package.json]: It is not such import, you can revert to `^`. I now semver is working, but with certain version packages, i'm more sure, that nothing break.

Comment (metainfa) [inline package.json]: Moreover yarn and now npm 5 lock packages.

**Comment by langpavel** [comment]
Hey @metainfa I'm trying.. not in my forces now. see #102, please... this is merged.. and much more.. time to relax this package to two: `SQL` template tag and temporary bridge to original `pg`

**Comment by langpavel** [comment]
BTW  @metainfa you now have ownership of `pg-async`. Be careful

**Comment by langpavel** [comment]
I have fresh https://github.com/langpavel/node-sql-fan

Goals: SQL template tags will be there.
New SqlBuilder class will be there. No magic, simply you saw this field and there. from this.. join this.. where and where... wery simle and useful

**Comment by langpavel** [comment]
Commits here are included in #102.

**Pull request closed without merging**